app/view_utils/money_view_utils.rb

```diff
@@ -0,0 +1,15 @@
+module MoneyViewUtils
+  extend ActionView::Helpers::NumberHelper
+
+  module_function
+
+  # Converts instance of Money to properly formatted and localized string for view purposes
+  # Replaces spaces with narrow non breaking spaces, we do not want monetary amounts to be split on separate lines
+  def to_humanized(m, locale)
```

Comment:
My reasoning was that this would take everything it could as parameters. And no global state would be used. 
Translations would be handled like a database. 

But it's not that big of a deal to switch the signature to e.g. 

```ruby
def to_humanized(m, locale = I18n.locale)
```
if it really improves calls to this.